src/templates/ApiEndpoint.tsx

```diff
@@ -532,6 +551,7 @@ export default function ApiEndpoint({ data, pageContext: { menu, breadcrumb, bre
                                             ? pathDescription(item)
                                             : item.description}
                                     </ReactMarkdown>
+                                    <Security item={item} objects={objects} />
                                     <Parameters item={item} objects={objects} />
```

Comment:
If we switched to sentence case in Parameters, which we have in the new Security heading, that'd be great
<img width="312" alt="Screenshot 2024-02-20 at 16 10 09" src="https://github.com/PostHog/posthog.com/assets/4550621/b2ce7a9f-5fe4-4865-a6c3-ca98aa70a043">